kind/README.md

```diff
@@ -0,0 +1,121 @@
+# Kind Deployment Guide
+
+This is a simplified deployment of Kuma and the Demo application on top of [kind](https://kind.sigs.k8s.io/). It follows the steps described in [kubernetes](../kubernetes/README.md) folder, replacing the Minikube deployment.
+
+## Prerequisites
+
+There are only two prerequisites required before you run the demo: [docker](https://docs.docker.com/get-docker/) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+## Running the demo
+
+Running the demo is easy, just type `./kind.sh` and the script will download, deploy and configure everything as described in [kubernetes](../kubernetes/README.md). In the end it will 
```

Comment:
`In the end it will` seems to end very abruptly

Reply:
Fixed, thanks!